# Mutation and Copy Number Variation Columns

## Single gene display

To display a single gene, enter that gene, choose your dataset and click 'done'. To see more than one gene, add another column.When displaying a single gene the gene model will be displayed at the top of the column. The gene model is a composite of all transcripts in Refseq with alternating light and dark shades of grey to show different exons. UTR regions are shorter in height than the CDS region. We display 2Kb upstream to show the promoter region. Use the column menu to toggle to show intronic regions.

## Chromosome range display

To display a genomic region, enter the genomic region, choose your dataset and click 'done'. We accept entire chromosomes \(e.g. chr1\), arms of chromosomes \(e.g. chr19q\), or chromosomes coordinates. Coordinates need to be in this format: chr1:100-4,000. When displaying a chromosome range, genes will be shown in dark blue at the top of the column. Hovering over a gene will display the gene name in the tooltip. Alt-click to freeze the tooltip to click on the gene name and learn more about that gene. Note that introns are always shown in this mode.

## Zooming

When displaying either a single gene or chromosome range you can click and drag on the gene\(s\) at the top to zoom into a region. Shift click on the gene\(s\) to zoom out. 

## Sorting

CNV data is sorted by the average of the entire column. Mutation data is sorted by the genomic position of the mutation. For both the CNV and mutation data, you can choose to instead sort by the region you are zoomed to through the column menu.

## Coloring

CNV is colored red to blue for amplification to deletion. Mutation data is colored by the functional impact. More information can be found here: [http://xena.ucsc.edu/how-we-characterize-mutations/](http://xena.ucsc.edu/how-we-characterize-mutations/)

## Examples

[Examples of CNV data](http://xena.ucsc.edu/segmented-cnv-data/)

[Examples of Mutation data](http://xena.ucsc.edu/positional-mutation-view/)



